WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverActivity.java

```diff
@@ -110,8 +110,11 @@ private void downloadExternalMedia() {
     }
 
     private boolean isAllowedMediaType(@NonNull Uri uri) {
```

Comment:
Great work @geriux 🙇 
Given that `isAllowedMediaType` has a `NonNull` annotation for the `uri` parameter I would suggest moving the null check on the two call sites instead (if statements on line 100 and 106 above). Wdyt?

Reply:
Thanks for the suggestion! I'll update it 🚀

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-Android/pull/19947/commits/e837837053d114602301ec68fec1907bd90579a8

Reply:
Thank you @geriux 🙇